libs/ts-rest/core/src/lib/client.ts

```diff
@@ -122,14 +132,12 @@ export function getRouteResponses<T extends AppRouter>(router: T) {
 export type AppRouteFunction<
   TRoute extends AppRoute,
   TClientArgs extends ClientArgs
-> = AreAllPropertiesOptional<
-  Without<DataReturnArgs<TRoute, TClientArgs>, never>
-> extends true
+> = AreAllPropertiesOptional<DataReturnArgs<TRoute, TClientArgs>> extends true
   ? (
-      args?: Prettify<Without<DataReturnArgs<TRoute, TClientArgs>, never>>
+      args?: Prettify<DataReturnArgs<TRoute, TClientArgs>>
     ) => Promise<Prettify<ApiRouteResponse<TRoute['responses']>>>
   : (
-      args: Prettify<Without<DataReturnArgs<TRoute, TClientArgs>, never>>
```

Comment:
Does this fix this comment? https://github.com/ts-rest/ts-rest/pull/182/files#r1119446639

Reply:
Nah, that was fixed in a separate PR. This is just cleaning up the type to reduce duplication. The `Without` part was moved into `DataReturnArgs`